google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -316,6 +316,73 @@ def _create_load_subparser(parser):
   return load_parser
 
 
+def _create_pipeline_subparser(parser):
+  pipeline_parser = parser.subcommand('pipeline', 'Load from GCS, query, and extract data from GCS '
+                                                  'into a BigQuery table. If creating a new table, '
+                                                  'a schema should be specified in YAML or JSON in '
+                                                  'the cell body, otherwise the schema is inferred '
+                                                  'from existing table.')
+
+  # common arguments
+  pipeline_parser.add_argument('-b', '--billing', type=int, help='BigQuery billing tier')
+  # TODO(rajivpb): I've naively made this a common param. Any objections?
+  pipeline_parser.add_argument('-nc', '--nocache', help='Don\'t use previously cached results',
```

Comment:
Does this make sense to specify in a pipeline? Even if it does, no-cache should be true by default in a pipeline, so a pipeline doesn't pick up state from some other run of some random other query.

Reply:
Understood. Removed this in https://github.com/googledatalab/pydatalab/pull/494 and have hard-coded this to False. At the time of coding this up, I hadn't completely dived into what this parameter meant, but it makes sense now.